plugins/importexport/native/filter/NativeXmlSubmissionFileFilter.inc.php

```diff
@@ -197,10 +197,10 @@ function handleRevisionElement($node, $stageId, $fileId) {
 					$submissionFile->setUserGroupId($userGroup->getId());
```

Comment:
We're iterating an arbitrary user's groups per the context where the import is happening.  This could result in multiple groups which have access to this stage, or none.  If multiple, we'll break on the first one we find.  If none, we'll select any group (well, the last one we found) which the user is a member of, even though it doesn't have access to the requested stage.